packages/xdl/src/Project.js

```diff
@@ -1792,7 +1792,7 @@ export async function startExpoServerAsync(projectRoot: string) {
         developerTool: Config.developerTool,
       });
     } catch (e) {
-      ProjectUtils.logDebug(projectRoot, 'expo', `Error in manifestHandler: ${e} ${e.stack}`);
+      ProjectUtils.logError(projectRoot, 'expo', e.stack);
```

Comment:
It seems like it'd be helpful to know what part of Expo CLI had a problem when it prints an error to the console -- does it make sense to log some more information here?

Reply:
I removed the prefix because the log output was something like this:
```
Error in manifest handler: XDLError: Error: ...
```
The "what part of Expo CLI" information should be clear from the included stack trace.